docs/architecture/gameUtil.md

```diff
@@ -0,0 +1,52 @@
+# GameUtil
+
+The [GameUtil](../../src/utils/gameUtil.ts) orchestrates
+loading the [GameState](../data/GameState),
```

Comment:
`../data/GameState` should be `../data/gameState.md`

Reply:
Fixed